client/lib/api/endpoints.dart

```diff
@@ -2,34 +2,49 @@
 
 class Endpoint {
   static const _whoMhPrefix = 'who-mh-';
+  static const _whoMh2Prefix = 'who-mh2-';
   static const _prodProjectId = 'who-mh-prod';
   static const _prodServiceUrl = 'https://covid19app.who.int';
 
+  static const _region = 'europe-west6';
+
   static bool _isProd(String projectId) {
     return projectId == _prodProjectId;
   }
 
   static String _projectIdShort(String projectId) {
-    if (!projectId.startsWith(_whoMhPrefix)) {
+    if (!projectId.startsWith(_whoMh2Prefix) &&
+        !projectId.startsWith(_whoMhPrefix)) {
       throw Exception(
           "Project: $projectId doesn't match prefix: $_whoMhPrefix");
     }
-    return projectId.substring(_whoMhPrefix.length);
+    return _isProd(projectId)
+        ? projectId.substring(_whoMhPrefix.length)
+        : projectId.substring(_whoMh2Prefix.length);
   }
 
   static String _serviceUrl(String projectId) {
+    if (_isProd(projectId)) {
```

Comment:
Don't worry about making a breaking change here. You can remove the old prod code and we'll just get it all working once the production project is exposed publicly.

Reply:
Sounds good, removed the prod endpoints for now and left a TODO above `isProd` to get this working once V2 is deployed.